test/ecdsatest.c

```diff
@@ -135,7 +135,7 @@ static int x9_62_test_internal(int nid, const char *r_in, const char *s_in)
     /* Use ECDSA_sign_setup to avoid use of ECDSA nonces */
     if (!TEST_true(ECDSA_sign_setup(key, NULL, &kinv, &rp)))
         goto x962_int_err;
-    if (!TEST_ptr(signature = ECDSA_do_sign_ex(digest, 20, kinv, rp, key)))
+    if (!TEST_ptr(signature = ECDSA_do_sign_ex(digest, SHA_DIGEST_LENGTH, kinv, rp, key)))
```

Comment:
I just noticed that this change (and many later ones) exceed the eighty character line length limit in the style guidelines.  They'll need reformatting:

```
    if (!TEST_ptr(signature = ECDSA_do_sign_ex(digest, SHA_DIGEST_LENGTH, kinv,
                                               rp, key)))
```

Reply:
ready